packages/theme/src/cli/services/dev.ts

```diff
@@ -99,7 +96,12 @@ export async function dev(options: DevOptions) {
     },
   }
 
-  const server = await setupDevServer(options.theme, ctx)
+  const {server, renderProgress} = await setupDevServer(options.theme, ctx)
+
+  const storefrontPassword = await storefrontPasswordPromise
+  session.storefrontPassword = storefrontPassword
+
+  await renderProgress()
```

Comment:
👍🏻 great idea!

How do you feel about a rename to clarify that this is unrelated to the `renderLinks` call below?

Reply:
Renamed to `renderThemeSyncProgress` 👍